src/app/GitExtUtils/GitUI/Theming/ColorHelper.cs

```diff
@@ -138,60 +142,16 @@ double DistanceTo(KnownColor c2)
             }
         }
 
-        /// <summary>
-        /// Roughly speaking makes a linear transformation of input orig 0 &lt; orig &lt; 1
-        /// The transformation is specified by how it affected a pair of values
-        /// (exampleOrig, oppositeOrig) to (example, opposite).
-        /// </summary>
-        private static double Transform(double orig,
-            double exampleOrig, double oppositeOrig,
-            double example, double opposite)
-        {
-            // if the value we are transforming was outside segment [example, opposite]
-            // then transform it with segment [0, example] or [opposite, 1] instead
-            if (oppositeOrig < exampleOrig && exampleOrig < orig)
-            {
-                oppositeOrig = 1d;
-                opposite = opposite < example ? 1d : 0d;
-            }
-            else if (orig < exampleOrig && exampleOrig < oppositeOrig)
-            {
-                oppositeOrig = 0d;
-                opposite = example < opposite ? 0d : 1d;
-            }
-
-            double deltaExample = example - opposite;
-            double deltaExampleOrig = exampleOrig - oppositeOrig;
-
-            // prevent division by zero
-            if (Math.Abs(deltaExampleOrig) < 0.01)
-            {
-                oppositeOrig = (oppositeOrig - 0.2).WithinRange(0, 1);
-                exampleOrig = (exampleOrig + 0.2).WithinRange(0, 1);
-                deltaExampleOrig = exampleOrig - oppositeOrig;
-            }
-
-            double deltaOrig = orig - oppositeOrig;
-
-            // delta / deltaOrig = deltaExample / deltaExampleOrig
-            double result = opposite + (deltaOrig / deltaExampleOrig * deltaExample);
-            return result.WithinRange(0, 1);
-        }
-
         /// <remarks>0.05 is subtle. 0.3 is quite strong.</remarks>
         public static Color MakeBackgroundDarkerBy(this KnownColor name, double amount) =>
-            ThemeSettings.InvariantTheme.GetNonEmptyColor(name)
-                .TransformHsl(l: l => l - amount)
-                .AdaptBackColor();
+            ThemeSettings.Theme.GetNonEmptyColor(name)
+                    .TransformHsl(l: l => l - amount);
 
-        public static bool IsLightTheme()
+        public static bool IsDarkTheme()
```

Comment:
Why not use [`Application.IsDarkModeEnabled`](https://learn.microsoft.com/dotnet/api/system.windows.forms.application.isdarkmodeenabled)?

Reply:
The short answer is that ThemeModule.IsDarkTheme cannot be used in Dashboard (where this is used).
I have mostly used Application.IsDarkModeEnabled for consequences of using the experimental dark mode, like overriding buttons to FlatStyle and ThemeModule.IsDarkTheme for other color relatded changes.
It is not exactly the same.

If the concept of theming is narrowed to light/dark for system colors (as WinForms supports) and custom for GE colors, then only Application.IsDarkModeEnabled could be used in all situations.
Then I would like to suppress WFO5001 globally too. I really hope that dark mode is not removed even if all quirks are not fixed for .net10 ((just two issues that I know of remains so GE could live with the current support.) Thanks to .NET engineers that fixed this in the final hours...